packages/desktop/src/renderer/components/Channel/CreateChannel/CreateChannel.test.tsx

```diff
@@ -118,7 +118,7 @@ describe('Add new channel', () => {
   it.each([
     ['   whitespaces', FieldErrors.Whitespaces],
     ['----hyphens', FieldErrors.Whitespaces],
-    ['!@#', ChannelNameErrors.WrongCharacter]
+    ['!@#general', 'Your channel will be created as']
```

Comment:
I think the right approach here is to provide a complete matching statement... I mean "Your channel will be created as #general"

Reply:
Thanks @siepra.  I actually decided to move this test from an error case to a success case in the previous test.  That seemed more appropriate.